audioprocessor.js

```diff
@@ -0,0 +1,37 @@
+class Processor extends AudioWorkletProcessor {
+    constructor(options) {
+        super(options);
```

Comment:
So another issue that I came across was that if I don't add 'options' in the constructor and in the call to super, it crashes when the polyfill is being used.
![image](https://user-images.githubusercontent.com/16245985/81136902-42ed9200-8f76-11ea-80c1-a436614722d4.png)